content/en/monitors/types/error_tracking.md

```diff
@@ -130,6 +130,22 @@ Error Tracking monitors use [Issue States][2] to ensure that your alerts stay fo
 
 **Ignored** issues are errors requiring no additional investigation or action. By marking issues as **Ignored**, these issues are automatically muted from monitor notifications.
 
+## Troubleshooting New Issue monitors
+As of September 2024, New Issue monitors are no longer time-based by default to help ensure no errors are missed. Previously, monitors created before this change included filters `issue.age` and/or `issue.regression.age`, which are no longer added by default. While not recommended, these filters can still be used if manually specified.
```

Comment:
This kind of note should be in a release note. Here we should describe how things work now.
```suggestion
```